### DISCUSSION TITLE
[Platform] Undo and redo feature

### DISCUSSION THREAD

**Discussion by lucagiraudo** [presenta-software/presenta-lib#43] [Ideas]
I deleted for mistake the slide i was creating (pressing DELETE on keyboard) and I didn't find a way to restore it. It could be useful to add a feature the let users undo their last action by pressing a button or cmd+z.
(or is it there and I didn't find it? 🤔 )

**Comment by abusedmedia** [comment]
It's definitely a must have feature not present yet.
